src/EnergyPlus/api/runtime.cc

```diff
@@ -53,13 +53,21 @@
 #include <EnergyPlus/StateManagement.hh>
 #include <EnergyPlus/UtilityRoutines.hh>
 
-void cClearAllStates() {
-    EnergyPlus::EnergyPlusData state;   //THIS IS TEMPORARY
-    EnergyPlus::clearThisState(state);
-    EnergyPlus::clearAllStates(state.outputFiles);
+EnergyPlusState stateNew() {
```

Comment:
The cClearAllStates function is now gone, and it is replaced with a full set of state management functions:
```
stateNew()
stateReset(state)
stateDelete(state)
```

This makes for much more intentional management of the simulation.

Reply:
For the uninformed (i.e. me), why do we need an explicit ```deleteState()``` method? Doesn't this have to be deleted on the client-side?